File: packages/python3/model/nitro_pack.py
class NitroPack:
    def __init__(self, json):
        self.id = json.get("id")
        self.x = json.get("x")
        self.y = json.get("y")
        self.z = json.get("z")
        self.radius = json.get("radius")
        self.nitro_amount = json.get("nitro_amount")
        self.respawn_ticks = json.get("respawn_ticks")

File: packages/ruby/model/nitro_pack.rb
class NitroPack
    attr_reader :id
    attr_reader :x
    attr_reader :y
    attr_reader :z
    attr_reader :radius
    attr_reader :nitro_amount
    attr_reader :respawn_ticks
    def initialize(json)
        @id = json["id"]
        @x = json["x"]
        @y = json["y"]
        @z = json["z"]
        @radius = json["radius"]
        @nitro_amount = json["nitro_amount"]
        @respawn_ticks = json["respawn_ticks"]
    end
end